spec/sisimai/smtp/error_spec.rb

```diff
@@ -13,6 +13,10 @@
   isntbounces = ['delivered', 'feedback', 'vacation']
   dependondsn = ['undefined', 'onhold']
 
+  dependondsn_hard = [
```

Comment:
__[rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `dependondsn_hard`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->